emsdk.py

```diff
@@ -2448,24 +2447,25 @@ def run_emcc(tools_to_activate):
         return
 
 
-def emscripten_cache_directory():
-  return os.path.join(emscripten_config_directory, ".emscripten_cache")
-
-
 # Copy over any emscripten cache contents that were pregenerated. This avoids
 # the user needing to immediately build libc etc. on first run.
 def copy_pregenerated_cache(tools_to_activate):
   for tool in tools_to_activate:
     pregenerated_cache = getattr(tool, 'pregenerated_cache', None)
     if not pregenerated_cache:
       continue
+    config = {}
+    for specific_cfg in tool.activated_config().split(';'):
+      name, value = specific_cfg.split('=')
+      config[name] = value.strip("'")
+    cache_root = os.path.join(config['EMSCRIPTEN_ROOT'], 'cache')
```

Comment:
I am slightly worried to not see a utility function for this (getting a key from the config in an activated config) - is there maybe another way to do this? or why has this never been needed before?

regardless it would be nice to put this in a utility function, so the code is easier to read i think.

Reply:
I found a way nicer solution which is just to remove this function :)